drivers/SPI.cpp

```diff
@@ -29,23 +29,25 @@ CircularBuffer<Transaction<SPI>, TRANSACTION_QUEUE_SIZE_SPI> SPI::_transaction_b
 #endif
 
 SPI::SPI(PinName mosi, PinName miso, PinName sclk, PinName ssel) :
-        _spi(),
+    _spi(),
 #if DEVICE_SPI_ASYNCH
-        _irq(this),
-        _usage(DMA_USAGE_NEVER),
-        _deep_sleep_locked(false),
+    _irq(this),
+    _usage(DMA_USAGE_NEVER),
+    _deep_sleep_locked(false),
 #endif
-        _bits(8),
-        _mode(0),
-        _hz(1000000),
-        _write_fill(SPI_FILL_CHAR) {
+    _bits(8),
+    _mode(0),
+    _hz(1000000),
+    _write_fill(SPI_FILL_CHAR)
+{
```

Comment:
This brace set feels like it should have an intentation.

Reply:
function body, no indentation needed